### DISCUSSION TITLE
Printing issue

### DISCUSSION THREAD

**Discussion by joostdecock** [freesewing/freesewing#7070] [Forum]
Hello, How do I print a pattern?. I have exported it using A4 setting but it doesn't download.

[![User false](https://backend3.freesewing.org/users/false/card)](https://freesewing.org/users/user?id=false)

**Comment by BenJamesBen** [comment]
If you successfully exported a pattern by using the "A4 PDF" button on the export page...
![Screenshot 2024-09-10 at 6 27 29 AM](https://github.com/user-attachments/assets/c672715d-cd0a-4528-9bcf-80c7ad0603fb)
... then your web browser should have downloaded a PDF file to your device. Please check on your device in the downloads folder or the place where your web browser normally downloads files to. You can then just print the file in the manner that you print other PDF files on your device.

If you need additional help, please provide more information about what device and web browser you are using, along with information about any error messages that you see.
